pkg/services/store/entity/sqlstash/sql_storage_server.go

```diff
@@ -224,14 +234,26 @@ func readEntity(rows *sql.Rows, r FieldSelectRequest) (*entity.Entity, error) {
 }
 
 func (s *sqlEntityServer) Read(ctx context.Context, r *entity.ReadEntityRequest) (*entity.Entity, error) {
+	ctx, span := s.tracer.Start(ctx, "storage_server.Read")
+	defer span.End()
+	ctxLogger := s.log.FromContext(ctx)
```

Comment:
We can do something like this instead of passing method (and key) each time:
```suggestion
	ctxLogger := s.log.FromContext(ctx).WithSuffix("method", "read", "key", r.Key)
```